mobile/src/common/enums/navigation/course-tabs-name.enum.ts

```diff
@@ -0,0 +1,7 @@
+enum CourseTabsName {
```

Comment:
And don't forget about filename 
```suggestion
enum CourseScreenName {
```